src/zenml/model/model_config.py

```diff
@@ -71,18 +73,19 @@ def _validate_recovery(
         if recovery:
             if not values.get("create_new_model_version", False):
                 logger.warning(
-                    "Using `recovery` flag without `create_new_model_version=True` makes no effect"
+                    "Using `recovery` flag without `create_new_model_version=True` has no effect."
                 )
         return recovery
 
     @validator("version")
     def _validate_version(
         cls, version: Union[str, ModelStages]
     ) -> Union[str, ModelStages]:
-        if isinstance(version, str) and version in ModelStages._members():
-            logger.warning(
-                f"Version `{version}` matches one of the possible `ModelStages`, if you want to fetch "
-                "model version by its' stage make sure to pass in instance of `ModelStages`."
+        if isinstance(version, str) and version in [
```

Comment:
Isn't this always a string?

```suggestion
        if version in [
```